### DISCUSSION TITLE
Thousands of bat detections per hour when no bats

### DISCUSSION THREAD

**Discussion by stewartwlewis** [rdz-oss/BattyBirdNET-Pi#49] [Q&A]
Hi, hope somebody can help point out what I am doing wrong.  No matter what I do I get thousands of bat detections per hour even when there are no bats and the recordings seem to have nothing in them.  When its daylight I get a much lower number of bird detections but the vast majority are from “foreign” birds.
I am using a Raspberry Pi (tried both RPI4 8GB and RPI5 4GB) connected to an AudioMoth USB microphone through a high quality 3m USB cable powered from a battery that can output 3a at 5v.
I have tried with both the downloadable OS install and "manually" Installed Bookworm 64bit light and downloaded the package from github.
Set up for my geographical location and to record both birds and UK bats with the BattyBirdNet AudioMoth noise reduction turned on (also tried off).
I updated the firmware on the AudioMoth USB to 1.3.1.  The switch on the side is in the Default setting and the LED flashes green. 
The microphone/cable are working fine through my laptop and when I listen to the Spectrogram BattyBirdnet
I initially tried recording in my garden but there is a road nearby and I blamed the thousands of detections on that or general house noise.  I moved it to my local woods, no buildings and the nearest road is about half a mile away.  No difference, still thousands of bat detections per hour.
Any suggestions would be most welcome and happy to supply any additional information that might help.
Stewart

**Comment by rdz-oss** [comment]
Hi @stewartwlewis , this sounds like there is likely a noise source in your area/setup that the neural network has not encountered before. Could you post an example of a false trigger (a picture of the spectrogram)? This would help to get an idea of what might be the root cause. Currently, the Bavaria classifier is trained with te most samples of noise. Does it behave the same? 

If it is noise , then it can be fixed by sending me some (30 or so) samples that I can use to train the network to ignore.


Would you kindly post a spectrogram?

Best regards,
Richard

**Reply by rdz-oss** [reply]
I was able to resolve similar issue of false detections before, e.g. in US and other UK setups. The classifier tests twice per second if there is noise or a bat. If it does not know the noise (e.g.  an insect or from electrical equipment, .. )  it could be triggered about 120 times per second leading to many thousand detections. Thankfully the neural network can learn new types of noise and hence this could be resolved so far with noise engineering. Lets see what is the cause in your case, could also be something with the setup. The spectrogram should show the type of noise we are dealing with. If it is a noise problem I will make you a file drop fro some samples. You can write me at battybirdnet - at - mailbox.org if you like.

**Reply by stewartwlewis** [reply]
Hi Richard
Thanks very much for your reply and help..  
I have attached a couple of spectrograms.  One from my house which has noise above 190kHz but I can't see anything else dated 06-09-2024 and one from the woods which has no noise - well nothing, dated 08-09-2024. I can't attached the WAV files but happy to send them whichever way is best for you. 
Although I clear down the file system each day as it fills up in two nights I saved a whole pile of the png and wav files and it's easy to get more!
In relation to system noise,  I have changed the power supply, including using battery, the  Rpi and cable.  The only common item is the AudioMoth, I only have one of those.

Thanks again
Stewart
![Common_noctule-100-2024-09-08-birdnet-00_01_00 wav](https://github.com/user-attachments/assets/c51987bb-10e7-4acd-bf1f-a3c53c21bbbe)
![Common_pipistrelle-100-2024-09-06-birdnet-23_05_00 wav](https://github.com/user-attachments/assets/baa12bec-cfd8-4bba-897c-d68fc17358b8)

**Reply by rdz-oss** [reply]
Hi Stewart,
thank you! Do you have some spectrograms where the filter is not active? The filter erases much of the noise pattern which might have triggered it. Hence, the noise shows up better in a spectrogram without the filter.
BR
Richard

**Comment by stewartwlewis** [comment]
Hi Richard
Sorry I should have thought of that.  Checked and all the recordings I saved have the filter active.  I will collect some more but it won't be after the weekend as I am away.  Also blowing a gale in the UK which is not good for recordings.
I just noticed a similar UK tread on the analyser github page, I will keep an eye on that one also.
Thanks again for your help..
Stewart

**Comment by stewartwlewis** [comment]
Hi Richard
Sorry for the delay, deployment problems and rubbish UK weather.  I don't think it actually needs a proper deployment as I get "hits" all the time so I put the Audiomoth microphone in a box filled with acoustic foam in my car in the garage. as it's the Quietest place I could think of.  Raspberry pi is powered by a battery outside the car.
I hve your noise reduction filter for the AudioMoth turned on.
These are some of the spectrographs I got.
![Lesser_noctule-92-2024-09-26-birdnet-05_11_15 wav](https://github.com/user-attachments/assets/6e9f5816-74f6-4d27-9e5a-565e7833a015)
![Brown_or_Grey_long_eared_bat-93-2024-09-26-birdnet-21_24_21 wav](https://github.com/user-attachments/assets/6005be0f-b78b-443a-a39a-6e0956a72603)
![Common_noctule-100-2024-09-25-birdnet-22_58_44 wav](https://github.com/user-attachments/assets/44bead93-af99-40ae-bc3b-1e544367b3bc)

**Comment by stewartwlewis** [comment]
Ho Richard
This is the spectrograph from the audiomoth in the acoustic foam showing a bat detection when I can see no change in the spectrograph that suggests anything to detect is this as it should be or have I got something fundamentally wrong.
Thanks again for helping me understand this.
Stewart
<img width="1592" alt="spectrograph common noctule 2024-10-01 205425" src="https://github.com/user-attachments/assets/791076d6-b67e-42b0-9957-d70c432b4a27">

**Comment by davidmam** [comment]
I note in the original post you have the switch in the default position. Should it not be in Custom?
